lib/tests/streamlit/color_picker_test.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018-2020 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""color_picker unit test."""
+
+from tests import testutil
+import streamlit as st
+from parameterized import parameterized
+
+
+class ColorPickerTest(testutil.DeltaGeneratorTestCase):
```

Comment:
Can we add tests for invalid values
- an integer
- a string that doesn't match the regex